source/write/delete.txt

```diff
@@ -0,0 +1,202 @@
+.. _kotlin-sync-write-delete:
+
+================
+Delete Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: remove, drop, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to remove
+documents from a MongoDB collection by performing **delete operations**.
+
+A delete operation removes one or more documents from a MongoDB collection.
+You can perform a delete operation by using the ``deleteOne()`` or
+``deleteMany()`` methods.
+
+.. TODO .. tip:: Interactive Lab
+
+..    This page includes a short interactive lab that demonstrates how to
+..    modify data by using the ``deleteMany()`` method. You can complete this
+..    lab directly in your browser window without installing MongoDB or a code editor.
+
+..    To start the lab, click the :guilabel:`Open Interactive Tutorial` button at the
+..    top of the page. To expand the lab to a full-screen format, click the
+..    full-screen button (:guilabel:`⛶`) in the top-right corner of the lab pane.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`.
+
+.. TODO To learn how to create a
+.. free MongoDB Atlas cluster and load the sample datasets, see the
+.. :ref:`<kotlin-sync-get-started>` tutorial.
+
+The documents in this collection are modeled by the following {+language+} data class:
+
+.. literalinclude:: /includes/write/delete.kt
+   :start-after: start-data-class
+   :end-before: end-data-class
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+Delete Operations
+-----------------
+
+You can perform delete operations in MongoDB by using the following methods:
+
+- ``deleteOne()``, which deletes *the first document* that matches the search criteria
+- ``deleteMany()``, which deletes *all documents* that match the search criteria
+
+Each delete method requires a **query filter** document, which specifies the
+search criteria that determine which documents to select for removal.
+For more information about query filters, see the 
+:manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+the {+mdb-server+} manual.
+
+Delete One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``deleteOne()`` method to remove a
+document in which the value of the ``name`` field is ``"Happy Garden"``:
+
+.. literalinclude:: /includes/write/delete.kt
+   :start-after: start-delete-one
+   :end-before: end-delete-one
+   :language: kotlin
+   :copyable:
+   :dedent:
```

Comment:
Intro here doesn't match the code sample (comment left in the code file)

Reply:
fixed